glean-core/android/src/test/java/mozilla/telemetry/glean/private/EventMetricTypeTest.kt

```diff
@@ -215,11 +215,8 @@ class EventMetricTypeTest {
         Glean.setUploadEnabled(false)
         assertEquals(false, Glean.getUploadEnabled())
         eventMetric.record(mapOf(testNameKeys.testName to "event2"))
-        try {
```

Comment:
Removing this and expecting the NPE means everything after that won't run, doesn't it?

Reply:
Oh, good catch.